docs/src/dingemans.md

```diff
@@ -169,32 +169,89 @@ legend_plot = plot(legend=:top, framestyle=:none, legendfontsize=11)
 for (i, (_, _, label, _, linestyle)) in enumerate(models)
     plot!(legend_plot, [], [], label=label, linestyles=linestyle, linewidth=2, color=i)
 end
-plot!(legend_plot, [], [], label="Bathymetry", color=:black,)
+legend_plot_bathymetry = plot(legend_plot, [], [], label="Bathymetry", color=:black,)
 
 xlims_zoom = [(-25, 15), (0, 40), (5, 45), (-100, -60)]
 snapshot_plots_zoom = [plot(snapshot_plots[i], xlims=xlims_zoom[i], ylims=(0.75, 0.85), title="Zoomed in at t = $(times[i])") for i in 1:4]
 
 # Combine all plots
-all_plots = [snapshot_plots..., legend_plot, snapshot_plots_zoom...]
+all_plots = [snapshot_plots..., legend_plot_bathymetry, snapshot_plots_zoom...]
 plot(all_plots...,
-    size=(900, 1100),
-    layout=@layout([a b; c d; e{0.14h}; f g; h i]),
+     size=(900, 1100),
+     layout=@layout([a b; c d; e{0.14h}; f g; h i]),
 )
 
-savefig("dingemans_comparison.png") # hide
+savefig("dingemans_temporal.png") # hide
 nothing # hide
 ```
 
-![Dingemans comparison](dingemans_comparison.png)
+![Dingemans temporal](dingemans_temporal.png)
 
 The results show how different dispersive wave models capture the wave evolution over the trapezoidal bathymetry.
 
+## Comparison with Experimental Data
+
+During the experiment of Dingemans, the wave evolution was recorded at six gauges along the flume. These measurements provide a reference for validating the numerical models.
+We compare the numerical results with the experimental data at the corresponding gauge locations.
+
+```@example dingemans
+path_dingemans = joinpath(@__DIR__, "data", "Dingemans", "Dingemans.csv")
```

Comment:
It will be easier to keep the data in this repository. However, it may be better to make them easily accessible to people using DispersiveShallowWater.jl (without having to work with the `docs` directory explicitly. What do you think about adding a function that returns the data and refers to Dingemans works in the docstring?

Such a function may read a CSV file - or just store the data in the function itself. I am not sure which approach will be better.